src/oneNotePicker.tsx

```diff
@@ -39,13 +38,14 @@ export class OneNotePicker extends OneNotePickerBase<OneNotePickerProps, OneNote
 		const { focusOnMount, ariaSelectedId } = globals;
 		const { recentSectionsExpanded } = this.state;
 
-		const notebookRenderStrategies: ExpandableNodeRenderStrategy[] = notebooks ?
-			notebooks.map(notebook => new NotebookRenderStrategy(notebook, globals)) : [];
-
-		const sharedNotebookRenderStrategies: ExpandableNodeRenderStrategy[] = sharedNotebooks ?
-			sharedNotebooks.map(sharedNotebook => new SharedNotebookRenderStrategy(sharedNotebook, globals)) : [];
+		// Sort both personal and shared notebooks by last modified time (last accessed time for shared notebooks)
+		const allNotebooks: (Notebook | SharedNotebook)[] = notebooks;
+		if (sharedNotebooks) {
+			allNotebooks.concat(sharedNotebooks);
+		}
+		allNotebooks.sort(this.sortNotebooksByLastModifiedTime);
```

Comment:
The sorting type should be an option in the picker props so we don't change existing behavior.

Reply:
I actually think this should be default behavior - there's no integration where this sorting shouldn't happen (at least teams and outlook)

Reply:
I agree with Jorge. If later we need the old ordering, we can add this in as a picker prop.